config/export.yaml

```diff
@@ -1166,7 +1166,7 @@ data:
   records:
   - instrument: LSSTCam-imSim
     name: i
-    abstract_filter: i
+    band: i
```

Comment:
I'm not sure these changes are helpful; I assume the registry needs to be updated anyway, and the script that does that will also remake the export file from scratch.

Reply:
I'm happy to leave this package to you to sort out. For many scenarios changing the export file is enough because the butler repo is recreated dynamically.